### PR TITLE
Add case studies template

### PR DESCRIPTION
**Pull request by mattbowen-usds**

### CODE REVIEW

File: tech-far-hub/gatsby-node.ts [hunk 1]
```diff
@@ -142,18 +144,20 @@ export const createPages: GatsbyNode["createPages"] = async ({ graphql, actions,
     }));
     const isIndex = node.pageName === "index";
     let parentPath: string;
-    if (node.pageName === "index") {
-      parentPath = pathParts.join("/");
-    } else {
-      parentPath = pathParts.slice(0, -1).join("/");
-    }
-    const parentPathRegex = `/${parentPath.replace("/", "\\/")}\\/.*/`;
+    parentPath = pathParts.slice(0, -1).join("/");
```
Comment (vim-usds) [inline tech-far-hub/gatsby-node.ts:147]: I appreciate how you're able to get around these limitations! Seems like one day I want research if there's an easier way. This node file is a keeper tho!

File: tech-far-hub/src/components/page-layout-nav.tsx [hunk 2]
```diff
@@ -12,6 +12,14 @@ interface IPageLayoutNav {
   siblings: {
     readonly nodes: readonly {
       readonly frontmatter: IMinimalFrontmatter | null;
+      readonly parent:
+        | {}
```
Comment (vim-usds) [inline tech-far-hub/src/components/page-layout-nav.tsx:16]: Assuming the pipe is an OR, if so, why is that leading OR needed in the definition of the readonly parent?

File: tech-far-hub/src/components/page-layout-nav.tsx [hunk 3]
```diff
@@ -39,28 +49,51 @@ const PageLayoutNav: React.FC<IPageLayoutNav> = ({
...
+        "relativeDirectory" in node.parent
+      ) {
+        const actualSlug = node.frontmatter.slug === "index" ? "" : node.frontmatter.slug;
+        if (isTopLevel) {
+          // If this is a top-level page, like Get Started, we actually want first-children
+          if (
+            node.parent.relativeDirectory.startsWith(pageContext.pathParts.join("/").slice(1)) &&
+            node.parent.relativePath !== pageContext.filePath
+          ) {
+            return <Link to={`/${node.parent.relativeDirectory}/${actualSlug}`}>{node.frontmatter.heading}</Link>;
```
Comment (vim-usds) [inline tech-far-hub/src/components/page-layout-nav.tsx:71]: this is great modularity, I'm learning so much!

**Review by vim-usds — COMMENTED** [review]
You're my goto Gatsby person!

File: tech-far-hub/src/components/page-layout-nav.tsx (hunk 2 above)
Comment (mattbowen-usds) [reply tech-far-hub/src/components/page-layout-nav.tsx:16]: You know, I don't know --- the formatted just adds it when I hit save. Imma have to figure that out...

**Pull request merged**